tools/atip/atip/web/web.py

```diff
@@ -452,6 +452,15 @@ def click_element_by_key(self, key, display=True):
             return True
         return False
 
+    def click_element_by_id_with_js(self, key, display=True):
```

Comment:
please add description of this method, explain why use js here to do click, but not direct element.js. Just because element.click will fail when VKB shelter the button, and js can avoid this issue. Then user can choose whether to use this method.